sentry-android-core/src/main/java/io/sentry/android/core/IDebugImagesLoader.java

```diff
@@ -0,0 +1,13 @@
+package io.sentry.android.core;
+
+import io.sentry.protocol.DebugImage;
+import java.util.List;
+import org.jetbrains.annotations.ApiStatus;
+
+/** Used for loading the list of debug images from sentry-native. */
+@ApiStatus.Internal
+public interface IDebugImagesLoader {
```

Comment:
`h`: Shouldn't this be in `io.sentry`?

Reply:
not needed, Java backend apps don't need it